### ISSUE TITLE
VPAT — Please Change/Remove Aria-Hidden Properties

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#3730]
Please do not use `aria-hidden="true"` properties on font-awesome icons used as control elements.  The `aria-hidden="true"` property makes the element invisible to screen readers, therefore completely disabling the control for screen reader users.

Where controls have no textual descriptions, (such as an edit or trash control), please replace `aria-hidden="true"` with an `aria-label` property containing clear and precise context.

This property is found in several twig files:
* [ ] autograding/AutoResults.twig
* [ ] course/CourseMaterials.twig
* [ ] forum/CategoriesForm.twig
* [ ] admin/users/StudentList.twig
* [ ] admin/users/GraderList.twig
* [ ] admin/PlagiarismHighLightingKey.twig
* [ ] grading/simple/Display.twig
* [ ] grading/simple/Details.twig
* [ ] grading/electronic/SubmissionPanel.twig
* [ ] submission/homework/CurrentVersionresults.twig
* [ ] submission/homework/BulkUploadBox.twig
* [ ] Homepage.twig
* [ ] Notifications.twig

This property is found in these PHP files:
* [ ] views/forum/ForumThreadView.php
* [ ] views/admin/PlagiarismView.php

**Comment by elihschiff** [comment]
I have never used a screen reader so I am sorry for the confusion. (also is there a recommended screen reader to test with?)
I am trying to solve this issue, I first went to the submitty home page and the "Change preferred names" button does have `aria-hidden="true"`, as you said, but that icon is inside a `<a>` tag which has `aria-label="Change preferred names"` Would the correct fix just be to change the inner icon to also have `aria-label="Change preferred names"`?

Here is what the live html looks like.
```html
<a class="icon" tabindex="0" href="javascript:userNameChange()" aria-label="Change preferred names" title="Change preferred names">
    <i class="fas fa-pencil-alt" aria-hidden="true"></i>
</a>
```
Here is what I think is how you want this fixed but it seems odd to me to have to repeat the same aria-label on the 2 elements inside each other.
```html
<a class="icon" tabindex="0" href="javascript:userNameChange()" aria-label="Change preferred names" title="Change preferred names">
    <i class="fas fa-pencil-alt" aria-label="Change preferred names"></i>
</a>
```

**Comment by pbailie** [comment]
We don't need to duplicate `aria-label` in the control.  Please add `aria-label` _**or**_ `aria-describedby` where one or the other doesn't already exist.  But if an `aria-label` or `aria-describedby` property already exists within the control, then removing `aria-hidden="true"` should be sufficient.

Please ensure that `aria-label` is very specific in purpose and `aria-describedby` is appropriately related to what is being used to describe the control.  e.g. `aria-label="Delete gradeable Lab04"` is much better than `aria-label="Click here to delete"`.

Thank you for helping.
